main.go

```diff
@@ -409,121 +403,6 @@ func SetLoggedIn(w http.ResponseWriter, r *http.Request, attemptCookie *LogInAtt
 	handle(err)
 }
 
-func employeeDashboard(w http.ResponseWriter, r *http.Request) {
-	http.ServeFile(w, r, "./templates/employeehomescreen.html")
-}
-
-func forgotEmail(w http.ResponseWriter, r *http.Request) {
-	http.ServeFile(w, r, "./templates/forgotemail.html")
-}
-
-func verifyEmailtoRecover(w http.ResponseWriter, r *http.Request) {
-	http.ServeFile(w, r, "./templates/verifyemailtorecover.html")
-}
-
-func postRecoveredEmail(w http.ResponseWriter, r *http.Request) {
-	http.ServeFile(w, r, "./templates/postrecoveredemail.html")
-}
```

Comment:
This part too ^ Also, during the development of your code, we switched to using `RenderTemplate` similar to these:
https://github.com/MonopolyTechnic/simple-banking-system/blob/main/main.go#L131
https://github.com/MonopolyTechnic/simple-banking-system/blob/main/main.go#L175